### PR TITLE
fix: Issue #45 - Values are not exceeding Boundaries

### PR DESCRIPTION
**Pull request by luckyman20**
@botraunak Please review it !!

### CODE REVIEW

**Comment by botraunak** [comment]
@luckyman20 Just removing the flex won't solve it! it will still go out of screen on lower screen size.
also add this
- Restrict the number of digits shown after decimal ( max 2)
- Look for css that would help adjust the numbers in card into a new line if less space

screen shot 2018-01-02 at 7 34 26 pm

**Comment by luckyman20** [comment]
@botraunak Updated the PR :-)

File: app/src/dashboard/dashboard.html [hunk 1]
```diff
@@ -48,11 +48,11 @@ <h1 class="md-display-2 text-center no-margin">{{ vm.dashboardData.shareAccounts
     <div layout="row">
         <md-card layout-gt-md="row" layout-align="center center" flex class="savings-total-card md-padding">
             <h1 class="display-1" flex>{{ 'label.heading.totalAccounts' | translate }}</h1>
-            <h1 class="display-1 text-right" flex="60">{{ vm.dashboardData.totalSavings }}</h1>
+            <h1 class="display-1 text-right" style="word-wrap:break-word;">{{ vm.dashboardData.totalSavings }}</h1>
         </md-card>
         <md-card layout-gt-md="row" layout-align="center center" flex class="loan-total-card md-padding">
             <h1 class="display-1" flex>{{ 'label.heading.totalLoan' | translate }}</h1>
-            <h1 class="display-1 text-right" flex="60">{{ vm.dashboardData.totalLoan }}</h1>
+            <h1 class="display-1 text-right" style="word-wrap:break-word;">{{ vm.dashboardData.totalLoan }}</h1>
```
Comment (botraunak) [inline app/src/dashboard/dashboard.html]: remove inline styles to the sass file, add a class name and I think it should be fine 👍

Comment (luckyman20) [reply app/src/dashboard/dashboard.html]: @botraunak Is 'display-1' class is usable?

Comment (botraunak) [reply app/src/dashboard/dashboard.html]: no display-1 is a genric class a something like card-amount and you can find sass files in the assets/stylesheets directory where you can your class in custom.scss

**Comment by luckyman20** [comment]
@botraunak I have tried customizing [this file](https://github.com/openMF/web-self-service-app/blob/master/app/assets/stylesheets/_custom.scss) but no use. Not showing anything. Can you please guide me in the right direction?

**Comment by botraunak** [comment]
@luckyman20 You're on the right direction you need to add your custom class here. however any changes won't auto reload the styles like it should. it's an error in the build process have opened the issue here. You would need to stop the `gulp serve` command and start it again after making changes to sass for now to test. [https://github.com/openMF/web-self-service-app/issues/64](https://github.com/openMF/web-self-service-app/issues/64)

**Comment by luckyman20** [comment]
@botraunak Please see the changes. According to me, these changes are enough but they are not working.

**Pull request merged**